### ISSUE TITLE
Does not work Phalcon\Validation\Validator\Alpha

### ISSUE THREAD

**Issue by jack-theripper** [phalcon/cphalcon#11386]
Check for alphabetic character for russian language.

Validator uses:

`if !ctype_alpha(value) {`

but it does not work with the Russian locale, in php big problem with setlocale on different operating systems.

how to change the condition in the code validator ?

For example

```php
if ((bool) preg_match('/[^[:alpha:]]/imu', $value)) {

// Field :field must contain only letters

} else {
//
}
```

**Comment by sergeyklay** [comment]
Can you please create PR with tests to the `2.1.x` branch?

**Comment by sergeyklay** [comment]
Fixed in 2.1.x branch
